aerospike/tests/test_aerospike.py

```diff
@@ -37,7 +39,14 @@ def _test_check(aggregator):
         'aerospike.set.stop_writes_count', 0, tags=['namespace:test', 'set:characters', 'tag:value']
     )
     aggregator.assert_metric('aerospike.namespace.tps.write', at_least=0)
-    aggregator.assert_all_metrics_covered()
+    try:
+        for _ in range(60):
+            aggregator.assert_metric('aerospike.namespace.latency.write_ops_sec')
+            if aggregator.metric_names:
+                break
+            time.sleep(1)
+    except Exception:
+        aggregator.assert_all_metrics_covered()
```

Comment:
How does that work?
The first time `aggregator.assert_metric` will fail and won't be retried.

Also why do we `aggregator.assert_all_metrics_covered()` in the except block?